docs/docs/keymaps/list-of-codes.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: List of Keycodes
```

Comment:
```suggestion
title: List of Codes
```
Was there a reason this was different? Also, not sure which one is better. "Keycodes" seems more descriptive and used through the docs but "Codes" is what it was always called in the title.

Reply:
I think I wanted to try it out since "Codes" seemed too vague for my liking and somewhat confused me when I first read through the docs. Do you think we should keep it as "Codes" or switch to "Keycodes" everywhere?

Reply:
I'd personally prefer "keycodes" too, with my only concern being term overlap with QMK. But that's likely not a significant issue. "Code" can mean a lot more things, especially confusable with source code.

Reply:
If we do change it, do we need the "List of" quantifier still, or would "Keycodes" be better?

Reply:
I intentionally minimize my knowledge of QMK, so I have no idea what you're referring to there. I like "List of" because it makes it clearer that it's a "reference" page rather than an "informative" page.

Reply:
All "codes" are now "keycodes".